Data/Map/Lazy.hs

```diff
@@ -88,24 +88,37 @@ module Data.Map.Lazy (
     -- * Map type
     Map              -- instance Eq,Show,Read
 
-    -- * Operators
-    , (!), (!?), (\\)
+    -- * Construction
+    , empty
+    , singleton
+    , fromSet
+
+    -- ** From Lists
+    , fromList
+    , fromListWith
+    , fromListWithKey
+    , fromAscList
+    , fromAscListWith
+    , fromAscListWithKey
+    , fromDistinctAscList
+    , fromDescList
+    , fromDescListWith
+    , fromDescListWithKey
+    , fromDistinctDescList
```

Comment:
Let's have three subsections for un-ordered, ascending, and descending.

Reply:
Done.